collections/ansible_collections/cloudkit/config_as_code/README.md

```diff
@@ -0,0 +1,225 @@
+# Ansible Collection - cloudkit.config_as_code
+
+This role defines the configuration as code of AAP to run the Ansible playbooks
+that are part of CloudKit.
+
+## Credentials
+
+Since AAP runs in an OpenShift cluster, we expect the required credentials to
+be passed as environment variables. These environment variables are defined as
+`Secrets` in OpenShift, and used by a container group.
+
+We define 2 container groups in AAP, one used to reconcile the configuration of
+AAP itself, and one used for cluster fulfillment operations.
+
+### Config-as-code environment variables
+
+In order to reconcile the configuration of AAP, we require the following
+variables:
+
+- `RH_USERNAME` and `RH_PASSWORD`: your Red Hat console username and password,
+  used to manage the subscription of AAP
+- `AAP_HOSTNAME`: URL of the AAP instance to be configured
+- `AAP_VALIDATE_CERTS`: true if the SSL certificate behind `AAP_HOSTNAME`
+  should be checked
+- `AAP_ORGANIZATION_NAME`: the AAP organization that should be created
+- `AAP_PROJECT_NAME`: the name of the project to be created
+- `AAP_PROJECT_GIT_URI`: the repository that is behind the AAP project
+- `AAP_PROJECT_GIT_BRANCH`: the git branch to use
+- `AAP_EE_IMAGE`: the registry URL to the execution environment image
+
+These variables must be defined in a secret named:
+`${AAP_ORGANIZATION_NAME}-${AAP_PROJECT_NAME}-config-as-code-ig` in the
```

Comment:
What does the `-ig` suffix mean?

Reply:
...ah, I guess that is "instance group".

Reply:
yep, I added that suffix, not sure if it makes sense for a secret name